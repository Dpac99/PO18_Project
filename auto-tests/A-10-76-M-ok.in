100015
1
e1.obj
0
