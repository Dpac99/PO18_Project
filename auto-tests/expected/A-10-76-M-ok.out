Identificador: Menu principal
1 - Abrir
2 - Guardar
3 - Portal Pessoal
5 - Portal do Estudante
6 - Portal do Delegado
0 - Sair
Escolha uma opção: Ficheiro a abrir: Resultados do inquérito do projecto P1 da disciplina D1
Pode preencher inquérito do projecto P5 da disciplina D1
Menu principal
1 - Abrir
2 - Guardar
3 - Portal Pessoal
5 - Portal do Estudante
6 - Portal do Delegado
0 - Sair
Escolha uma opção: 